src/config.json

```diff
@@ -3,7 +3,7 @@
   "staticFilePath": "enter the pathway to static binary file with BGP data",
   "outlierDetectionAlgorithm": "mad",
   "madParameter": "enter the sensitivity parameter",
-  "prefix": "enter the prefix or prefixes of choice here",
+  "prefix": "151.101.0.0/16,63.135.66.0/23",
   "asn": "enter the ASNs of interest here",
```

Comment:
Future PRs ought to handle this filter and the peerIp filters. We probably want to consider just how expressive this needs to be (e.g., if prefix + peer is set in config, are they treated as two totally separate filters, or one combined filter?). This is good for this PR, just something to keep in mind for the near future.